core/src/main/java/tc/oc/pgm/flag/LegacyFlagBeamMatchModule.java

```diff
@@ -50,6 +53,13 @@ public LegacyFlagBeamMatchModule(Match match) {
     this.beams = new HashMap<>();
   }
 
+  private boolean shouldShowBeams(Flag flag) {
```

Comment:
Is this unused? did you mean to call this from trackFlag?

Reply:
Ah this is my bad, forgot to remove this and didn't end up using it.

Reply:
Resolved in new commit.